Test/Integration Test/cypress/integration/PageObjects/objects.js

```diff
@@ -121,53 +121,6 @@ class Order
         
         }
 
-    subscription_product()
```

Comment:
It seems we have removed several cases, in order to avoid the authentication.

We should still tests all these cases, and inject the configuration values into the config.json from GitHub secrets, in order to make sure they run accordingly.

Am I missing something?

Reply:
@emicha We have created another ticket to handle subscription product.